packages/story-editor/src/components/colorPicker/basicColorPicker.js

```diff
@@ -138,6 +242,16 @@ function BasicColorPicker({
           <StyledPlus />
         </StyledButton>
       </Body>
+      {showDialog && (
+        <ConfirmationDialog
```

Comment:
@barklund I moved this over here currently but it's not really working -- the confirmation dialogue and the color picker popup can not be open at the same time. Wondering what would be a good approach for this.

One option could be moving the `ConfirmationDialog` to the `Color` input directly, to the same component that includes the `ColorPicker`, closing the color picker for the time of opening the confirmation dialogue, and once the user has interacted with the dialogue, reopening the color picker again. This could potentially tangle the Color input and the ColorPicker quite a bit. But maybe it's ok.

Not sure if there's a simpler alternative. Let me know if you have any alternative thoughts.